app/views/provider_interface/provider_users/index.html.erb

```diff
@@ -2,10 +2,9 @@
 
 <h1 class="govuk-heading-xl">Users</h1>
 
-<%= link_to(
+<%= govuk_button_link_to(
```

Comment:
There's also `Delete user` in `provider_users/show.html.erb`

Reply:
Fixed up here https://github.com/DFE-Digital/apply-for-teacher-training/pull/2688/commits/47f24401712baa5758f4f6c9924a5316f9b97939#diff-4fc2354ff570c561338be2d352678508R23